_includes/k8s/bestpractices.md

```diff
@@ -37,7 +36,11 @@ To generate a unique password, a technique such as the following, which
 generates a pseudo-random 24 character string, may be used:
 
 ```bash
-cat /dev/urandom | tr -dc 'a-zA-Z0-9-!@#$%^&*()_+~' | fold -w 24 | head -n 1
+# Generate strong password
+PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9-!@#$%^&*()_+~' | fold -w 24 | head -n 1)
+
+# Convert password to base64 representation for embedding in a K8S secret
+BASE64PASSWORD=$(echo $PASSWORD | base64)
```

Comment:
should be `echo -n` otherwise it will add an endline and break the password.

Reply:
Good catch! Fixed.